content/en/02-installing-pixie/03-install-guides/03-airgap-pixie.md

```diff
@@ -0,0 +1,281 @@
+./dev_dns_updater --domain-name="dev.withpixie.dev"  --kubeconfig=$HOME/.kube/config --n=plc
+```
+
+4. Navigate to `dev.withpixie.dev` in your browser. Make sure that the network you are on can access your cluster.
+
+### Authentication using Kratos / Hydra
+
+Self-managed Pixie Cloud only supports one organization.
+
+1. To setup the default admin account, check the logs for the `create-admin-job` pod by running:
+
+```bash
+kubectl logs create-admin-job-<pod_string> -n plc
+```
+
+2. Open the URL from the pod's logs to set the password for the `admin@default.com` user.
+
+<Alert variant="outlined" severity="warning">
+  If you've visited dev.withpixie.dev before, make sure to clear the cookies for this site or you'll get a login error.
+</Alert>
+
+3. Once the password has been set, login using `admin@default.com` for the `identifier` and your new password.
+
+<Alert variant="outlined" severity="warning">
+  There is a known issue with login on self-managed Pixie Cloud on Safari and Firefox. For now, use Chrome.
+</Alert>
+
+### Serve the Script Bundle
+
+After logging into the Pixie UI using the admin account, you will see a blank UI with the following errors in the DevTools console:
+
+<svg title='' src='airgap/ui-error.png' />
+
+To resolve these errors, you'll need to do one of the following:
+
+- [Setup the script dev environment](/installing-pixie/install-guides/airgap-pixie/#deploy-pixie-cloud-setup-the-script-dev-environment)
+- [Override the `SCRIPT_BUNDLE_URLS` environment variable](https://github.com/pixie-io/pixie/issues/485#issuecomment-1191969058)
+
+#### Setup the Script Dev Environment
+
+To setup the script dev environment:
+
+```
+
+git clone https://github.com/pixie-io/pixie.git
+cd pixie/src/pxl_scripts
+make dev
+
+```
+
+Open Chrome’s DevTools console and run the following:
+
+```
+
+localStorage.setItem('px-custom-oss-bundle-path', 'http://127.0.0.1:8000/bundle-oss.json')
+localStorage.setItem('px-custom-core-bundle-path', 'http://127.0.0.1:8000/bundle-oss.json')
+
+```
+
+Once you have set these variables, do a **soft** reload of the UI webpage (a hard reload will clear the variable you just set).
+
+Once you're able to see Pixie's UI, you'll need to use Pixie's UI to [create a deploy key](/reference/admin/deploy-keys/#create-a-deploy-key-using-the-live-ui). Record this value somewhere, we'll use it in a future step.
+
+### Invite others to your organization (optional)
+
+Add users to your organization to share access to Pixie Live Views, query running clusters, and deploy new Pixie clusters. For instructions, see the [User Management & Sharing](/reference/admin/user-mgmt) reference docs.
+
+## Deploy Pixie
+
+1. Download the vizier artifacts:
+
+```
+curl https://storage.googleapis.com/pixie-dev-public/vizier/latest/vizier_yamls.tar | tar x
+cd yamls
+```
+
+2. Update the `deploy-key` and `PX_CLUSTER_NAME` values in the `vizier/secrets.yaml` file. Remember that you previously created the deploy key in Pixie UI's.
```

Comment:
Maybe we can link to the step which explains how they can create the deploy key? Just in case they missed that step earlier.